src/components/plots/ProductConcentrationPlot.tsx

```diff
@@ -125,7 +133,30 @@ const ProductConcentrationPlot: React.FC<ProductConcentrationPlotProps> = ({
                 color: AGENT_AB_COLOR,
             },
         },
+        shapes: [] as Partial<Plotly.Shape>[],
```

Comment:
Instead of noting types here, could you just add a type annotation to `layout`? (e.g. `layout: Partial<Plotly.Layout> = ...`